Testscripts/Linux/verify_vmbus_interrupts.sh

```diff
@@ -57,16 +62,18 @@ function verify_vmbus_interrupts() {
                 fi
             done
         fi
-    done < "/proc/interrupts"
+    done < interrupts
 
-    if [ $nonCPU0inter -ne "$cpu_count" ]; then
-        LogErr "Not all CPU cores are processing VMBUS interrupts!"
+    if [ $nonCPU0inter -eq 0 ]; then
```

Comment:
For the small and mid-size VM, if there are some cpus not processing VMBUS interrupts, we just judge if $nonCPU0inter equal 0 will pass this error.
Could we judge $nonCPU0inter for two situations, one is small,mid-size VM, another is big VM?

Reply:
OK, that is fair question, if you send me your suggestion I am happy to look into. As I said in the Description, I look for more help to build the accurate verification.

Reply:
@lubaihua33 what is the range to determine big size VM vs. small/mid size VM?

Reply:
Proposed new way in 3856d2a42b7a05478982cff13599b1120e403e9b. @lubaihua33 ?